src/Symfony/Component/Serializer/Encoder/CsvEncoder.php

```diff
@@ -27,24 +27,28 @@ class CsvEncoder implements EncoderInterface, DecoderInterface
     const ESCAPE_CHAR_KEY = 'csv_escape_char';
     const KEY_SEPARATOR_KEY = 'csv_key_separator';
     const HEADERS_KEY = 'csv_headers';
+    const FORMULAS_START_CHARACTERS = array('=', '-', '+', '@');
```

Comment:
I don't think this needs to be exposed. So it could be relevant as a private const, that would work as of 4.0 only. If we accept the PR as a bugfix, this would not work since using constants for arrays is possible since PHP 5.6 only, whereas our 3.* branches support PHP 5.5).